project.clj

```diff
@@ -0,0 +1,10 @@
+(defproject com.github.liquidz/antq "99.9.9"
+  :dependencies [[org.clojure/clojure "1.10.3"]
+                 [org.clojure/data.xml "0.2.0-alpha6"]
+                 [org.clojure/data.zip "1.0.0"]
+                 [org.clojure/tools.cli "1.0.206"]
+                 [org.clojure/tools.deps.alpha "0.11.922"]
+                 [cheshire/cheshire "5.10.0"]
+                 [clj-commons/clj-yaml "0.7.106"]
+                 [version-clj/version-clj "2.0.1"]
+                 [rewrite-clj/rewrite-clj "1.0.644-alpha"]])
```

Comment:
Is this `project.clj` file required?
To install antq locally, we can use `make install`.

Reply:
> To install antq locally, we can use make install.

Nice one!